Goobi/src/org/goobi/production/flow/statistics/hibernate/UserDefinedFilter.java

```diff
@@ -325,11 +325,6 @@ protected void setStepDone(Integer exactStepDone) {
 			this.exactStepDone = exactStepDone;
 		}
 
-		@SuppressWarnings("unused")
-		private Boolean getCriticalQuery() {
-			return flagCriticalQuery;
-		}
-
```

Comment:
The method `setCriticalQuery` is currently used in another file. Removing that method and the member variable `flagCriticalQuery` should be addressed separately.